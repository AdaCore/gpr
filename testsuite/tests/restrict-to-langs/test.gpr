with "gpr2";
project Test is
   for main use ("main.adb");

   package Compiler is
      for Switches ("Ada") use ("-gnatwu", "-g", "-O1");
   end Compiler;
end Test;
