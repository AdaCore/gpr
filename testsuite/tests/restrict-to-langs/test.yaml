description:
    restricting autoconfiguration to gives languages
driver: build_and_run
project_file: test
main: main
