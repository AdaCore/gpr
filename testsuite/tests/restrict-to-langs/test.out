Restrict to existing language:
warning: no compiler driver defined for language 'Fortran'
Restrict to missing language:
prj1.gpr: error: no language for the projects tree due to language restriction
